src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SecureGroovyScriptTest.java

```diff
@@ -75,7 +75,7 @@
      * Basic approval test where the user doesn't have RUN_SCRIPTS privs but has unchecked
      * the sandbox checkbox. Should result in script going to pending approval.
      */
-    @Test public void basicApproval() throws Exception {
+    @Ignore("Upgrade issue; fix after next core update") @Test public void basicApproval() throws Exception {
```

Comment:
I would vote to print such messages on a separate line. I partially agree with Jesse's code style, but this long text pollutes the method signature IMO